firmware/defmt-itm/src/lib.rs

```diff
@@ -20,6 +20,9 @@ use core::{
 
 use cortex_m::{interrupt, itm, peripheral::ITM, register};
 
+#[cfg(armv6m)]
+compile_error! {"`defmt-itm` cannot be used on Cortex-M0(+) chips, because it requires an ITM peripheral"}
```

Comment:
```suggestion
compile_error!("`defmt-itm` cannot be used on Cortex-M0(+) chips, because it requires an ITM peripheral");
```

Reply:
My linter told me to use `{`, which I also didn't understood, but it actually also worked in some dummy code I tried.

Reply:
Ah, maybe you forgot the trailing `;`? I generally use `{ }` if there's multi-line content inside, but `();` if not.